doc/en/getting-started.rst

```diff
@@ -185,6 +185,21 @@ You can find out what kind of builtin :ref:`fixtures` exist by typing::
 
     pytest --fixtures   # shows builtin and custom fixtures
 
+Contributing tests to an existing project
```

Comment:
To be honest I would rather remove this entire session... it seems distracting and very narrow for a specific user case. I understand that was your use case, but nowadays IMHO people are used to virtualenvs in general (that includes venv and conda), so this in the end reads confusing to me, specially on the getting-started session.